ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -324,9 +324,12 @@ def delete_buckets(bucket_prefix, hours):
     aws = AWS()
     buckets_to_delete = aws.get_buckets_to_delete(bucket_prefix, hours)
     logger.info(f"buckets to delete: {buckets_to_delete}")
+    filename = "s3_bucket_deletion_list.txt"
```

Comment:
What location are you intend to store this file? This should be somewhere in logs dir I guess.
For what reason we exactly need this file to exist when those buckets suppose to be deleted?

Reply:
What location are you intend to store this file? This should be somewhere in logs dir I guess. ->
the location is `~`, I copied the code from here:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/cleanup/aws/cleanup.py#L458


For what reason we exactly need this file to exist when those buckets suppose to be deleted?
I want to check who buckets deleted. I cant see it via `log.info`